kraken/invoke/command.py

```diff
@@ -1,17 +1,16 @@
 import subprocess
 import logging
+import sys
 
 
 # Invokes a given command and returns the stdout
-def invoke(command):
+def invoke(command, timeout=None):
```

Comment:
should we set this to 60 by default. Might be helpful for not having to find all invoke commands and it'll allow commands to not get stuck waiting for a response.

Reply:
We are using the invoke function for calling the pod scenarios i.e powerfulseal commands which have their own timeout set, so if we set the default to 60 here, it will kill those commands. Setting this to None will allow us to define timeouts explicitly for each of the commands depending on the use case. Thoughts?

Reply:
Ah makes sense! Definitely don't want to add that for the powerfulseal calls